packages/e2e/src/index.ts

```diff
@@ -138,24 +138,38 @@ async function runTasksConcurrently(tasks: ((worker: TestWorker) => Promise<void
   }
 }
 
-function assertBrowserName(browserName: string): asserts browserName is 'webkit'|'chromium'|'firefox' {
+type Browser = "chromium" | "webkit" | "firefox"
+
+function assertBrowserName(browserName: string): asserts browserName is Browser {
   if (browserName !== 'firefox' && browserName !== 'chromium' && browserName !== 'webkit')
     throw new Error(`Unknown browser: ${browserName}`);
 }
 
-function configForTestSuite(suite: JestSuite) {
-  let config = {};
+interface UserConfig {
+  browsers: Browser[]
+  launchOptions: playwright.LaunchOptions,
```

Comment:
launch takes place before we have specific test suites. Jest passes us a GlobalConfig in the constructor of our test runner. That contains a rootDir property, and we can look inside that rootDir for the main playwright config. Then we can use that to retrieve the launch options.

Reply:
Thanks!